alacritty_terminal/src/ansi.rs

```diff
@@ -1,60 +1,4 @@
 //! ANSI Terminal Stream Parsing.
 
+#[doc(inline)]
 pub use vte::ansi::*;
```

Comment:
What does this attribute do? Probably shouldn't just export `*` here?

Reply:
It just inlines the docs for module.

The `*` was there before, but I can try to re-export things needed.